### ISSUE TITLE
Multi-task view: The keyboard navigation and the selected border for a window don't work very well

### ISSUE THREAD

**Issue by megatux** [elementary/gala#415]
Multi-task view: The keyboard navigation and the "selected" white border for the current window don't work very well.
Not always display the _current_ window. Also when using keyboard to move between windows it not always change visually (but it does internally, hard to explain). It also worked this way in previous 0.4 version. Can't take screenshots in multi-task view :man_shrugging: . I'll try to create a screencast.

**Comment by Blast-City** [comment]
Yeah, his needs a new design.  

You have to remember which window was in foreground before the workspace overview appears and then press in the opposite direction with the scroll keys.

Another weird thing is that the current window selected have different indication when you select with the keyboard (silver outline) and with the mouse (circle in the top left corner)

**Comment by TheRock892** [comment]
the window selection via the keyboard works in the multitasking view SUPER + ⬇️

but it does not work in the view of all windows that is activated with SUPER + A

and not even in the windows view of the current workspace with SUPER + W

**Comment by peteruithoven** [comment]
I'm closing this issue. There are 2 many issues grouped together in this issue:
1. Not always display the current window
2. When using keyboard to move between windows it not always change visually

There was also no screenrecording provided making it hard to follow. 

@Blast-City and @TheRock892 You added new issues which makes this issue even harder to follow.

**Comment by peteruithoven** [comment]
Feel free to open new, more focused issues.
